code/modules/power/apc.dm

```diff
@@ -826,43 +825,95 @@
 		return
 	if(src.z != 1)
 		return
-	src.occupier = new /mob/living/silicon/ai(src,malf.laws,null,1)
-	src.occupier.adjustOxyLoss(malf.getOxyLoss())
-	if(!findtext(src.occupier.name,"APC Copy"))
-		src.occupier.name = "[malf.name] APC Copy"
+	occupier = new /mob/living/silicon/ai(src,malf.laws,null,1) //DEAR GOD WHY?
```

Comment:
faint screaming
the ai is also the ai CORE because fuck, I don't know, everything is horrifyingly badly designed in ai code